### DISCUSSION TITLE
提示 prompt at: http://127.0.0.1:8300

### DISCUSSION THREAD

**Discussion by mtt-konan** [NateScarlet/auto-derby#283] [Q&A] [answered]
版本auto-derby-1.20.1
ocr到这个界面就停止了，能正常上网，用的uu加速器，page_example测试过正常。提示的好像就最后四行，其他的都是正常输出，保险起见调试文件我还是放在最后了

![q5](https://user-images.githubusercontent.com/102540842/161579118-5f0c0bf6-c969-4684-9911-06bb05010b93.png)

![q4](https://user-images.githubusercontent.com/102540842/161581367-37e670f7-b99d-4c6b-8af6-298a3cdd3758.png)

[problem.txt](https://github.com/NateScarlet/auto-derby/files/8411095/problem.txt)

**Comment by NateScarlet** [comment, marked as answer]
网页让你选对应的物品啊 你选不了吗？

[FAQ](https://github.com/NateScarlet/auto-derby/wiki/FAQ#%E8%84%9A%E6%9C%AC%E5%81%9C%E4%BD%8F%E4%BA%86)
